AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/EditorEntityVector3Property.py

```diff
@@ -0,0 +1,19 @@
+import azlmbr.math as math
+from editor_python_test_tools.editor_component.EditorEntityComponentProperty import EditorEntityComponentProperty
+from editor_python_test_tools.editor_entity_utils import EditorComponent
+
+
+class EditorEntityIntegerProperty(EditorEntityComponentProperty):
```

Comment:
update: Looks like the typename for this should be changed to `EditorEntityVector3Property`

Reply:
yep. I scrambled to get this up to a PR for others but made some copypaste errors along the way

Reply:
No worries, let me know when things are updated and I can re-review 👍